torch/_torch_docs.py

```diff
@@ -152,6 +152,28 @@ def merge_dicts(*dicts):
     tensor([ 1.2294,  2.2004,  1.3690,  1.7298])
 """.format(**common_args))
 
+add_docstr(torch.acosh,
+          r"""
+acosh(input, out=None) -> Tensor
+
+Returns a new tensor with arccosh (inverse hyperbolic cosine) of the elements of :attr:`input`.
```

Comment:
I'm not sure if we want to have **Returns a new tensor with _arccosh_**? Should this be _hyperbolic arccosine_?

Reply:
I think the part in parenthesis makes it clear enough

Reply:
Great, thanks @albanD! :+1: